src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -79,13 +79,22 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
     return index;
   }, [activeIndex, labels.length]);
 
+  const tablistRef = React.useRef<HTMLUListElement>(null);
```

Comment:
`tabListRef` unless its a valid single word?

Reply:
I had `tabListRef` but then I thought it would make sense as a single word, same as the tablist aria role.